src/NServiceBus.AcceptanceTests/Core/Hosting/When_endpoint_starts.cs

```diff
@@ -47,5 +47,17 @@ public MyEndpoint()
                    .EnableStartupDiagnostics();
             }
         }
+
+        class MyMessageHandler : IHandleMessages<MyMessage>
```

Comment:
Would it make sense to finally turn this test into an approval test that looks at the content of the file (potentially stripping out and normalizing unstable things such as datetime? it feels off that we don't approve what is currently in the diagnostics

Reply:
Agree, not sure how to deal with all the unstable stuff though so I'll do that in a separate PR